runtime/src/main/scala/tailcall/runtime/service/SchemaGenerator.scala

```diff
@@ -45,7 +45,7 @@ object SchemaGenerator {
         None,
         inputValue.name,
         toCalibanType(inputValue.ofType),
-        inputValue.defaultValue.map(DynamicValueUtil.toInputValue),
+        inputValue.defaultValue.map(DynamicValueUtil.toInputValue(_).get),
```

Comment:
`.get` are unsafe. Can we handle this more gracefully?

Reply:
Done.